src/DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,17 @@ public function getConfigTreeBuilder()
 
         $this->addBundlesSection($root);
 
+        $root
+            ->children()
+                ->arrayNode('ivory_ckeditor')
+                    ->addDefaultsIfNotSet()
+                    ->canBeEnabled()
+                    ->children()
+                        ->scalarNode('config_name')->defaultValue('cmf_sonata_phpcr_admin_integration')->end()
```

Comment:
The only think we use in here is a key. That key referes to a configuration on IvoryCkEditorBundle. You see it in a WIP PR on sandbox to use this one here (tried it local with this branch): https://github.com/symfony-cmf/cmf-sandbox/pull/363